operator/src/main/resources/scripts/startServer.sh

```diff
@@ -188,12 +206,109 @@ checkEnv \
   AS_SERVICE_NAME || exitOrLoop
 
 trace "LOG_HOME=${LOG_HOME}"
+trace "DATA_HOME=${DATA_HOME}"
 trace "SERVER_OUT_IN_POD_LOG=${SERVER_OUT_IN_POD_LOG}"
 trace "USER_MEM_ARGS=${USER_MEM_ARGS}"
 trace "JAVA_OPTIONS=${JAVA_OPTIONS}"
+trace "KEEP_DEFAULT_DATA_HOME=${KEEP_DEFAULT_DATA_HOME}"
+trace "LINK_SERVER_DATA_DIR=${LINK_SERVER_DATA_DIR}"
```

Comment:
Since this env var isn't supported in production, can we rename it?
EXPERIMENTAL_LINK_SERVER_DEFAULT_DATA_DIR